bin/reth/Cargo.toml

```
@@ -86,6 +86,7 @@ hex = "0.4"
 thiserror = { workspace = true }
 pretty_assertions = "1.3.0"
 humantime = "2.1.0"
+parse-display.workspace = true
```

Small nit (I think `cargo add` just adds `<crate>.workspace = true`, but I find the other more human readable):
```suggestion
parse-display = { workspace = true }
```